x-pack/plugins/observability/public/embeddable/slo/overview/slo_embeddable_factory.ts

```diff
@@ -42,6 +45,15 @@ export class SloOverviewEmbeddableFactoryDefinition implements EmbeddableFactory
     }
   }
 
+  public getPanelPlacementSettings: IProvidesPanelPlacementSettings<
+    SloEmbeddableInput,
+    unknown
+  >['getPanelPlacementSettings'] = () => {
+    const width = 8;
+    const height = 7;
```

Comment:
I guess that's internal to the dashboard system 👍🏻

Reply:
Yep dashboard system uses a grid system. I listed [here](https://github.com/elastic/kibana/issues/170710) the default values that are being used and I kind of picked these values as a starting point, getting inspired by the [links embeddable](https://github.com/elastic/kibana/blob/main/src/plugins/links/public/embeddable/links_embeddable_factory.ts#L72).